lib/engine/step/dividend.rb

```diff
@@ -48,6 +48,18 @@ def dividend_options(entity)
         end
       end
 
+      def variable_share_multiplier(_corporation)
```

Comment:
Could you add comments to all three of these functions, describing what they do and when they would need to be overridden?